package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "project-athena-build-uploader",
-  "version": "1.0.0",
+  "version": "0.4.0",
```

Comment:
???? why change the version ?

Reply:
Don't bully dependabot. He doesn't know any better.

Reply:
ah I see, its the lock file we're looking at. Makes sense. Thanks @dependabot